### PR TITLE
[WEB-17] Add Integration and IntegrationId entries in enums

### PR DESCRIPTION
**Pull request by Deacon-McIntyre**
In conjunction with these PRs: https://github.com/MindscapeHQ/raygun/pull/14999 | https://github.com/MindscapeHQ/Raygun.API.SDK/pull/51

This PR implements create-integration functionality within the Javascript SDK.

Screenshot of alert created using the SDK, then fetched in the app:
image

### CODE REVIEW

**Review by QuantumNightmare — APPROVED** [review]
:+1:

**Review by sumitramanga — APPROVED** [review]

File: lib/entities/alerts/enums.ts [hunk 1]
```diff
@@ -22,10 +22,12 @@ export enum AlertConditionType {
 
 export enum AlertActionType {
   User = "User",
+  Integration = "Integration",
```
Comment (sumitramanga) [inline lib/entities/alerts/enums.ts:25]: You will need to bump the SDK version inside the package.json if this is a separate deployment or if you want to merge into Crystals branch you won't need to bump the version in this branch :)

**Review by sumitramanga — CHANGES_REQUESTED** [review]
I left out piece to cover but otherwise your changes look good!!

File: lib/entities/alerts/enums.ts (hunk 1 above)
Comment (Deacon-McIntyre) [reply lib/entities/alerts/enums.ts:25]: Thanks for pointing this out! Have changed the base to Crystal's branch 👍

**Pull request merged**